packages/opc-styles/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@one-platform/opc-styles",
+  "version": "0.0.1",
+  "description": "This packages contains the all common styles which are important to get #### started with any component which includes global variables, colors, common mixin",
+  "main": "opc-styles.scss",
+  "scripts": {
+    "test": "./node_modules/.bin/node-sass -w ./test/ -o css/"
+  },
+  "author": "",
+  "license": "ISC",
```

Should we change it to MIT? It's almost the same as ISC, but it will just make it consistent with all our other components and repos.

```suggestion
  "license": "MIT",
```